apps/api/src/coscrad-cli/ingest-media-items.cli-command.ts

```diff
@@ -255,17 +263,22 @@ export class IngestMediaItemsCliCommand extends CliCommandRunner {
             }
         );
 
-        const createResourceFsas = createMediaItemFsas
-            /**
-             * We only want to create corresponding media resources when the MIME Type
-             * corresponds to a audio, video, or photograph.
-             */
-            .filter(({ payload: { mimeType } }) => isMediaResourceMimeType(mimeType))
-            .map((createMediaItemFsa, index) => {
-                const idToUse = generatedIds[index + partialPayloads.length];
-
-                return buildCreateResourceFsaForMediaItem(createMediaItemFsa.payload, idToUse);
-            });
+        const createResourceFsas = shouldCreateResources
```

Comment:
This code is getting quite difficult to read and reason about. Let's do some clean up. We could potentially make it far enough to include the `tags` option if we do this now.